torch/_jit_internal.py

```diff
@@ -994,22 +1037,6 @@ def _qualified_name(obj) -> str:
     return module_name + "." + name
 
 
-# Thin wrapper around SourceRangeFactory to store extra metadata
```

Comment:
Is this code block just moved or moved+modified? 

I think this refactor (along with others you made in this PR) makes sense, just that in general, it is slightly better to put refactoring in a separate PR whenever possible so that each PR has a clear purpose.

Reply:
it's just moved.